packages/xo-server-usage-report/report.html.tpl

```diff
@@ -121,6 +121,10 @@
       border:1px solid #95a5a6;
       text-align: center;
     }
+
+    .allResources table {
+      width: 1000px
+    }
```

Comment:
Why this change?
I prefer we use *min-with* or *max-width* instead of *width* and *em*s instead of *px*s.

Reply:
I changed the width of the table because all resources' stats have more information than the other stats.
If i use min-width or max-width, the tables will not have the same width. Consequently, it will downgrade the look of the report.

Reply:
Sorry my bad, it's better to use min-width for flexibility.